src/lib/override/adopted-probes.ts

```diff
@@ -376,6 +403,23 @@ export class AdoptedProbes {
 		}
 	}
 
+	private async deleteDuplicates (currentAdoptedIp: string, connectedProbe: Probe) {
+		const duplicates = await this.sql(ADOPTED_PROBES_TABLE)
+			.where({ ip: currentAdoptedIp })
+			.orWhere({ ip: connectedProbe.ipAddress })
+			.orWhere({ uuid: connectedProbe.uuid })
+			// First item will be preserved, so we are prioritizing probes with a custom city and online probes.
+			.orderByRaw(`isCustomCity DESC, lastSyncDate DESC, onlineTimesToday DESC, FIELD(status, 'ready') DESC`)
```

Comment:
I initially thought about considering tags here as well, but now I'm thinking: for this case to happen, the user must have adopted the same probe twice. It makes most sense to simply use his most recent data, doesn't it? If the user, e.g., didn't set a custom city the second time, it would be weird if the city suddenly reappeared. So maybe just `date_updated DESC, date_created DESC` ?

Reply:
I think it is possible that the adoption with earlier `date_created` syncs and becomes actual, and `date_updated` may change for many reasons (e.g. right now it is 21.11.2024 for all probes). As I see, we can remove `isCustomCity DESC` part and achieve the same behaviour.

Reply:
OK.